nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise 'NotImplementedError'
```

Comment:
- [ ] Let's try to make the exception message more descriptive so we can also tell which class has failed to implement the `correct_name` method.

  ```
  raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
  ```